src/jit/instr.h

```diff
@@ -86,13 +86,32 @@ enum GCtype : unsigned
     GCT_BYREF
 };
 
-// TODO-Cleanup:  Move 'insFlags' under _TARGET_ARM_
+#if defined(_TARGET_XARCH_)
+enum insFlags: uint8_t
+{
+    INS_FLAGS_None = 0x00,
+
+    INS_FLAGS_ReadsFlags = 0x01,
+
+    INS_FLAGS_WritesFlags = 0x02,
+
+    INS_FLAGS_x87Instr = 0x04,
+
+    //  TODO-Cleanup:  Remove this flag and its usage from _TARGET_XARCH_
+    INS_FLAGS_DONT_CARE = 0x00,
+};
+#elif defined(_TARGET_ARM_) || defined(_TARGET_ARM64_)
```

Comment:
IMO, you should maintain the `// TODO-Cleanup:  Move 'insFlags' under _TARGET_ARM_` here, since ARM64 shouldn't be using this (but does).

Reply:
Fixed.